.env.example

```
@@ -0,0 +1 @@
+ENV_NAME="LOCAL"
```

To follow POSIX Git convention, always add an empty line at the end of a file

```suggestion
ENV_NAME="LOCAL"

```